Docs/test_docs.py

```diff
@@ -0,0 +1,145 @@
+"""
```

Comment:
maybe we can just call this doc_diff or something? otherwise we might think that test_docs performs other checks such as those performed by pytest on the example snippets

Reply:
That is good suggestion. I renamed it.